lib/support/resultsStorage/index.js

```diff
@@ -15,7 +15,7 @@ function getDomainOrFileName(input) {
   return domainOrFile;
 }
 
-module.exports = function(input, timestamp, outputFolder, resultBaseURL) {
+module.exports = function(input, timestamp, outputFolder, resultBaseURL, resultBaseDir) {
```

Comment:
I'd prefer to keep the two path related parameters next to each other in the parameter list, and keep the URL last.